src/bytesToUuid.js

```diff
@@ -14,28 +14,28 @@ function bytesToUuid(buf, offset) {
   const bth = byteToHex;
 
   // join used to fix memory issue caused by concatenation: https://bugs.chromium.org/p/v8/issues/detail?id=3175#c4
```

Comment:
Did you double-verify that the above mentioned issue is no longer present? Either remove the comment or revert to the previous join-solution.